posnums.c

```diff
@@ -0,0 +1,67 @@
+#include <stdio.h>
+/* Asks for two positive integers, gives the everage of the two, states if they are prime, and if they are relatively prime.*/
```

Comment:
your logic is a bit off. It returns 4 as a prime number, sometimes doesn't print whether the numbers are relatively prime or not (11 and 1).